diff --git a/README.md b/README.md
index cf7249f..8815099 100644
--- a/README.md
+++ b/README.md
@@ -1 +1,2 @@
-Source for the html adventure game I'm building.
\ No newline at end of file
+Source for Dark Passenger, an interactive fiction.
+
diff --git a/data/loc.yaml b/data/loc.yaml
index 340fb50..a04d4d8 100644
--- a/data/loc.yaml
+++ b/data/loc.yaml
@@ -7,7 +7,7 @@ intro:
 
     by Jethro Larson
 
-    > [Dark beginning](#living-room)
+    > [Continue...](#living-room)
 
 #######
 
@@ -15,7 +15,7 @@ intro:
   title: Living Room
   content: |
     The morning sun gave me hope that today would be warm, but there's no 
-    comfort to be found in this little peasants home. 
+    comfort to be found in this little farm house. 
     [Whispers of wind](#living-room/whispers) seem to creak through the 
     house but the air is still, hanging with a 
     [thick stench](#living-room/smell).
@@ -108,7 +108,7 @@ intro:
         content: |
           "Lord what are we dealing with here?" I ask, directing my question at Jin.
         
-          <p><code>"<a href="#demonicon">The Demonicon</a> is your friend.</code>"<%cache.demonicon_on="true";game.inventory.update;%>
+          <p><code>"<a href="#demonicon">The tome</a> is your friend.</code>"<%cache.demonicon_on="true";game.inventory.update;%>
         
         
 ###################crap below
@@ -197,4 +197,10 @@ bedroom:
     There's a portrait on the wall of some ancestor, it's very old. The painting seems to look at you.
     EXAMINE shows close up and VIEW BACK option. The back shows the back of the person. There is otherwise nothing special.
 
-    There's a small table on one side of the room. On it is some incense that you can pick up. DP says he hates jasmine as you pick it up. Obove the table is a mirror facing the door. EXAMINE Shows the reflection. In it is a strange doll with a damaged arm. You can grab the doll through the mirror.
\ No newline at end of file
+    There's a small table on one side of the room. On it is some incense that you can pick up. DP says he hates jasmine as you pick it up. Obove the table is a mirror facing the door. EXAMINE Shows the reflection. In it is a strange doll with a damaged arm. You can grab the doll through the mirror.
+    
+---------------------------
+  
+tome:
+  title: Tome of Spirits
+  content:
\ No newline at end of file
diff --git a/static/css/global.css b/static/css/global.css
index b109392..33c5fb2 100644
--- a/static/css/global.css
+++ b/static/css/global.css
@@ -4,16 +4,16 @@ a:hover{color: #AF5763;text-decoration:underline;}
 
 code{color:#8F77EF;}
 
-html,body{height:100%;}
+html,body{min-height:100%;}
 body{
-  background:#111; font: 16px georgia, serif;
+  background:#111; font: 16px/1.4 georgia, serif;
   color: #999;}
 body.loading *{cursor: wait;}
 #page{
-  width: 550px; height:100%;
+  width: 550px; min-height:100%;
   margin:0 auto; padding: 24px;box-sizing:border-box;
   background: #191919;}
-  
+  #content{min-height: 600px;}
 dialog{
   display:none;
   position: absolute;top:80px; left: 50%;
diff --git a/static/js/game.js b/static/js/game.js
index 0b1d2e7..ce8c242 100644
--- a/static/js/game.js
+++ b/static/js/game.js
@@ -1,96 +1,93 @@
 /* Utils */
-function lookupOb(ob,arr){
+function lookupOb(ob, arr) {
 	var result = ob;
-	for(var i=0;i<arr.length;i++){
+	for (var i = 0; i < arr.length; i++) {
 		result = result[arr[i]];
-		if(result === undefined){
+		if (result === undefined) {
 			return undefined;
 		}
 	}
 	return result;
 }
 
-
 /*: Game object #constructor */
 var game = {
-	title:"Dark Passenger",
-	$title:$("#title"),
-	$content:$("#content"),
-	$dialogs : $("#dialogs"),
-	$locations : $("#locations"),
-	$inventory : $("#inventory"),
-	$people:$("#people"),
-	locations:{},
-	people:{},
-	inventory:{},
-	location:{
-		current:null
+	title: "Dark Passenger",
+	$title: $("#title"),
+	$content: $("#content"),
+	$dialogs: $("#dialogs"),
+	$locations: $("#locations"),
+	$inventory: $("#inventory"),
+	$people: $("#people"),
+	locations: {},
+	people: {},
+	inventory: {},
+	location: {
+		current: null
 	},
-	init: function(){
-		if(cache.location){
+	init: function() {
+		if (cache.location) {
 			game.location.render(cache.location);
-		}else{
-			game.newGame(); 
+		} else {
+			game.newGame();
 		}
 		game.addEvents();
 	},
-	newGame: function(){
+	newGame: function() {
 		cache.clear();
 		game.location.render("intro");
 	}
 };
 
-game.loadContent = function(locationName,lookupArray){
-	if(cache.location!=locationName){
-		game.location.render(locationName,lookupArray);
-	}else{
-		var data = lookupOb(game.location.current,lookupArray);
+game.loadContent = function(locationName, lookupArray) {
+	if (cache.location != locationName) {
+		game.location.render(locationName, lookupArray);
+	} else {
+		var data = lookupOb(game.location.current, lookupArray);
 		Dialog(data);
 	}
 };
 
-game.addEvents = function(){
-	$("#content, #dialogs").delegate("a","click",function(){
+game.addEvents = function() {
+	$("#content, #dialogs").delegate("a", "click", function() {
 		var $this = $(this);
-		if($this.hasClass("next")){
-			
-		}else if($this.hasClass("prev")){
-			
-		}
-		else if(this.hash){
+		if ($this.hasClass("next")) {
+
+		} else if ($this.hasClass("prev")) {
+
+		} else if (this.hash) {
 			var lookupArray = this.hash.slice(1).split("/");
-			var match = (/^#([^\/]+)(?:\/([^\/]+))*/).exec(this.hash);//Parse object lookups.
+			var match = (/^#([^\/]+)(?:\/([^\/]+))*/).exec(this.hash); //Parse object lookups.
 			var locationName = lookupArray[0];
-			return game.loadContent(locationName,lookupArray.slice(1));//Security hole?
+			return game.loadContent(locationName, lookupArray.slice(1)); //Security hole?
 		}
 	});
 };
 
-game.location.render=function(name,lookupArray){
-	if(name){
+game.location.render = function(name, lookupArray) {
+	if (name) {
 		game.location.current = game.locations[name];
-		cache.location=name;
+		cache.location = name;
 		//TODO handle lookupArray
 	}
 	var loc = game.location.current;
-	document.title = loc.title+" - "+game.title;
+	document.title = loc.title + " - " + game.title;
 	game.$title.text(loc.title);
 	game.$content.html(game.parseContent(loc.content));
 	game.renderMenu("locations");
 	game.renderMenu("people");
 };
-game.renderMenu = function(name){
-	HTML ="";
-	$.each(game[name],function(k,v){
-		if(v.visible){
-			HTML = '<li><a href="#'+k+'">'+v.name+'</a></li>';
+game.renderMenu = function(name) {
+	HTML = "";
+	$.each(game[name], function(k, v) {
+		if (v.visible) {
+			HTML = '<li><a href="#' + k + '">' + v.name + '</a></li>';
 		}
 	});
-	game['$'+name].html(HTML);
+	game['$' + name].html(HTML);
 };
 
-
-game.parseContent = function(txt){
+game.parseContent = function(txt) {
 	txt = tmpl(txt)(game);
 	txt = converter.makeHtml(txt);
 	return txt;
@@ -101,14 +98,14 @@ var cache = localStorage;
 
 /* Dialog
 **********/
-function Dialog(data){
-	var content = data.content; 
-	if($.isArray(data.content)){
-		content = content[0];//TODO Make paging functional
+function Dialog(data) {
+	var content = data.content;
+	if ($.isArray(data.content)) {
+		content = content[0]; //TODO Make paging functional
 	}
 	content = game.parseContent(content)
 	var $dialog = $("#zoom");
-	$dialog.find("h2").text(data.title?data.title:'');
+	$dialog.find("h2").text(data.title ? data.title : '');
 	$dialog.find("content").html(content);
 	$dialog.fadeIn(300);
 }
@@ -116,25 +113,25 @@ function Dialog(data){
 var converter = new Showdown.converter();
 
 //JSONP load function
-function location_load(data){
+function location_load(data) {
 	game.locations = data;
 }
 
-$(function(){
-	game.init();//TODO enable loading from cookie/session
-	var $dialog = $("#zoom").each(function(){
+$(function() {
+	game.init(); //TODO enable loading from cookie/session
+	var $dialog = $("#zoom").each(function() {
 		var $this = $(this);
 		var $content = $this.find(".content");
-		var $prev = $this.find(".prev").click(function(){
+		var $prev = $this.find(".prev").click(function() {
 			$this.trigger("prev");
 			return false;
 		});
-		var $next = $this.find(".next").click(function(){
+		var $next = $this.find(".next").click(function() {
 			$this.trigger("next");
 			return false;
 		});
-		$this.find(".close").click(function(){
+		$this.find(".close").click(function() {
 			$dialog.hide();
 		});
 	});
-});
\ No newline at end of file
+});
diff --git a/static/js/global.js b/static/js/global.js
deleted file mode 100644
index e69de29..0000000
diff --git a/views/game.html b/views/game.html
index 36de6c3..508a071 100644
--- a/views/game.html
+++ b/views/game.html
@@ -18,15 +18,13 @@
       <div id="content">{%block canvas%}{%endblock%}</div>
       <menu>
         <div id="inventory">
-          <script type="text/tmpl" id="inventory_tmpl">
-          <%if(cache.demonicon_on==="true"){%>
-            <a href="#demonicon">The Demonicon</a>
-          <%}%>
-          </script>
         </div>
         <div id="locations"></div>
         <div id="people"></div>
       </menu>
+      <footer>
+        <a href="#newGame" onclick="game.newGame();return false;">New Game</a>
+      </footer>
       <div id="dialogs">
         <dialog id="zoom">
           <header>
